app/code/community/Philwinkle/Fixerio/Model/Import.php

```diff
@@ -2,7 +2,7 @@
 
 class Philwinkle_Fixerio_Model_Import extends Mage_Directory_Model_Currency_Import_Abstract
 {
-    protected $_url = 'https://api.fixer.io/latest?base=%1$s&symbols=%2$s';
+    protected $_url = 'https://data.fixer.io/api/latest?access_key=YOUR_ACCESS_KEY&base=%1$s&symbols=%2$s';
```

Comment:
ACCESS_KEY needs to be an admin-configurable setting or have a valid fallback value. This should be store configuration-scoped. Please provide a setup script for that and we'll get this merged! Thanks!